packages/slice-machine/src/features/onboarding/OnboardingProgressStepper.tsx

```diff
@@ -71,11 +74,16 @@ export const OnboardingProgressStepper = () => {
                   onSelect={() => showStep(step)}
                   description={step.description}
                   completed={isCompleted}
+                  readOnly={step.readonly}
                   endAdornment={
-                    <Icon name="chevronRight" size="small" color="grey11" />
+                    step.readonly !== true && (
+                      <Icon name="chevronRight" size="small" color="grey11" />
+                    )
                   }
                 >
-                  {step.title}
+                  {isOnboardingCardVisibilityExperiment
+                    ? `${index + 1} ${step.title}`
+                    : step.title}
```

Comment:
```suggestion
                  {`${isOnboardingCardVisibilityExperiment ? `${index + 1} ` : ''}${step.title}`}
```

Reply:
Honestly, I find your suggested version much harder to read